samples/unity-of-bugs/Assets/Scripts/SentryNativeBridge.m

```diff
@@ -0,0 +1,121 @@
+#import <Sentry/Sentry.h>
+
+void SentryNativeBridgeAddBreadcrumb(const char* timestamp, const char* message, const char* type, const char* category, int* level) {
+    NSLog(@"Native Bridge: Adding breadcrumb.");
+    
+    if (timestamp == NULL && message == NULL && type == NULL && category == NULL && level == NULL) {
+        NSLog(@"Native Bridge: Breadcrumb empty. Dropping it.");
+        return;
+    }
+    
+    [SentrySDK configureScope:^(SentryScope * _Nonnull scope) {
```

Comment:
`m`: When you add `NS_ASSUME_NONNULL_BEGIN` below the header imports and `NS_ASSUME_NONNULL_END`, you don't need the `_Nonnull` specifiers as `_Nonnull` is default.